demos/digit-prediction/digits_prediction.nlogo

```diff
@@ -0,0 +1,527 @@
+extensions [ py ]
+
+globals [ path coords_x coords_y color_list digit]
+
+to setup-py
+  clear-all
+  py:setup py:python
+  (py:run
+    "import numpy as np"
+    "from keras.models import load_model"
+  )
+  ;; load model
+  py:run "model = load_model(\"model.h5\")"
+
+  ;; create coords list, useful to make color_list
+  set coords_x (range max-pxcor)
+  set coords_y (range max-pycor)
```

Comment:
I believe these will miss the last column/row, since `range` is exclusive on the end. They should be, e.g., `range (max-pxcor + 1)`. Also, you might want to start them at (e.g.) `min-pxcor` to be more robust to world sizes. It looks to me like the y-range is flipped, since the y-coordinate is the reverse of row numbers. Is it correctly distinguishing 6s from 9s right now? (can't test ATM) I believe it should be `(range max-pycor (min-pycor - 1) -1)`, unless I'm missing something.